docs/nimservice.md

```diff
@@ -129,6 +131,47 @@ Apply the manifest:
 kubectl create -f test-pod.yaml -n nim-service
 ```
 
+#### Example 2: 
+
+Create a file, `verify-pod-2.yaml`, with contents like the following example:
+
+```yaml
+---
+apiVersion: v1
+kind: Pod
+metadata:
+  name: verify-streaming-chat-2
```

Comment:
can we rename it as `verify-chat-completions` instead

Reply:
also, it would be good to include a section for perf analysis using the GenAI perf tool: https://docs.nvidia.com/deeplearning/triton-inference-server/user-guide/docs/client/src/c%2B%2B/perf_analyzer/genai-perf/README.html

Reply:
@shivamerla Done

Reply:
lgtm, thanks @angudadevops